bolero/environment/optimum_trajectory.py

```diff
@@ -117,13 +130,14 @@ def get_outputs(self, values):
         """
         if self.t == 0:
             values[:self.n_task_dims] = np.copy(self.x0)
-            values[self.n_task_dims:-self.n_task_dims] = np.zeros(
-                self.n_task_dims)
-            values[-self.n_task_dims:] = np.zeros(self.n_task_dims)
+            values[self.n_task_dims:2*self.n_task_dims] = np.zeros(self.n_task_dims)
+            if not self.calcAcc:
+                values[-self.n_task_dims:] = np.zeros(self.n_task_dims)
         else:
             values[:self.n_task_dims] = self.X[self.t - 1]
-            values[self.n_task_dims:-self.n_task_dims] = self.Xd[self.t - 1]
-            values[-self.n_task_dims:] = self.Xdd[self.t - 1]
+            values[self.n_task_dims:2*self.n_task_dims] = self.Xd[self.t - 1]
```

Comment:
Keep the use of whitespaces consistent, see PEP8:

> However, in a slice the colon acts like a binary operator, and should have equal amounts on either side (treating it as the operator with the lowest priority). In an extended slice, both colons must have the same amount of spacing applied. Exception: when a slice parameter is omitted, the space is omitted.

> Yes:
> 
> ham[1:9], ham[1:9:3], ham[:9:3], ham[1::3], ham[1:9:]
> ham[lower:upper], ham[lower:upper:], ham[lower::step]
> ham[lower+offset : upper+offset]
> ham[: upper_fn(x) : step_fn(x)], ham[:: step_fn(x)]
> ham[lower + offset : upper + offset]
> 
> No:
> 
> ham[lower + offset:upper + offset]
> ham[1: 9], ham[1 :9], ham[1:9 :3]
> ham[lower : : upper]
> ham[ : upper]

E.g.: 
`values[self.n_task_dims : 2*self.n_task_dims] = self.Xd[self.t-1] `